crates/rpc/rpc-engine-api/src/error.rs

```diff
@@ -88,9 +89,18 @@ pub enum EngineApiError {
     EngineObjectValidationError(#[from] EngineObjectValidationError),
     /// If the optimism feature flag is enabled, the payload attributes must have a present
     /// gas limit for the forkchoice updated method.
-    #[cfg(feature = "optimism")]
-    #[error("Missing gas limit in payload attributes")]
-    MissingGasLimitInPayloadAttributes,
+    // #[cfg(feature = "optimism")]
+    // #[error("Missing gas limit in payload attributes")]
+    // MissingGasLimitInPayloadAttributes,
+    #[error("0")]
```

Comment:
```suggestion
    #[error(transparent)]
```

Reply:
Changing this would error in:
```
the method `as_dyn_error` exists for reference `&Box<dyn ToRpcError>`, but its trait bounds were not satisfied
the following trait bounds were not satisfied:
`Box<dyn ToRpcError>: StdError`
which is required by `Box<dyn ToRpcError>: AsDynError<'_>`
`&Box<dyn ToRpcError>: StdError`
which is required by `&Box<dyn ToRpcError>: AsDynError<'_>`
`dyn ToRpcError: Sized`
which is required by `dyn ToRpcError: AsDynError<'_>` [E0599]
```
 besides error enums use `error(0)` such as https://github.com/paradigmxyz/reth/blob/ce1e401d217215dfdbe6c9de6b87efadf9f804e6/crates/rpc/rpc/src/eth/error.rs#L121-L123

Reply:
can you try `"{0}"` instead?

Reply:
```suggestion
    #[error("{0}")]
```